mage/query-modules/python/kmeans-clustering.md

```diff
@@ -0,0 +1,171 @@
+{children}
+</span>
+);
+
+[![docs-source](https://img.shields.io/badge/source-node2vec-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/main/python/kmeans.py)
+
+## Abstract
+
+The KMeans algorithm clusters given data by trying to separate samples in `n` groups of equal variance, by minimizing a criterion known as 
+within-the-cluster sum-of-squares. To understand more about it, jump to [algorithm](../../algorithms/machine-learning-graph-analytics/kmeans-clustering-algorithm) page.
+
+| Trait               | Value                                                          |
+| ------------------- | -------------------------------------------------------------- |
+| **Module type**     | <Highlight color="#FB6E00">**module**</Highlight>              |
+| **Implementation**  | <Highlight color="#FB6E00">**Python**</Highlight>              |
+| **Graph direction** | <Highlight color="#FB6E00">**directed/undirected**</Highlight> |
+| **Edge weights**    | <Highlight color="#FB6E00">**weighted/unweighted**</Highlight> |
+| **Parallelism**     | <Highlight color="#FB6E00">**sequential**</Highlight>          |
+
+## Procedures
+
+### `get_clusters( n_clusters, embedding_property, init, n_init, max_iter, tol, algorithm, random_state)`
+This procedure returns for each node to which cluster it belongs to.
+
+#### Input:
+
+- `n_clusters : int` ➡ The number of clusters to form.
+- `embedding_property : str` ➡ From which property of node will embeddings be read.
+- `init : str` ➡ Method for initialization. If `k-means++` is selected, then initial cluster centroids  are selected using sampling based on an empirical probability distribution of the points’ contribution to the overall inertia. This technique speeds up convergence, and is theoretically proven to be `O(logk)`-optimal. 
+If `random` then choose `n_clusters` observations (rows) at random from data for the initial centroids.
+- `n_init : int` ➡ Number of time the k-means algorithm will be run with different centroid seeds
+- `max_iter : int` ➡ Length of one walk in walk sampling.
+- `tol : float` ➡ Relative tolerance with regards to Frobenius norm of the difference in the cluster centers of two consecutive iterations to declare convergence.
+- `algorithm : str` ➡ Options are `lloyd`, `elkan`, `auto`, `full`
+- `random_state : int` ➡ Use an int to make the randomness deterministic
+
+#### Output:
+
+- `node: mgp.Vertex` ➡ Vertex for which we got cluster_id
+- `cluster_id: mgp.Number` ➡ Corresponding cluster id of vertex
+
+#### Usage:
+
+```cypher
+ CALL kmeans.get_clusters(2, "embedding", "k-means++", 10, 10, 0.0001, "auto", 1) YIELD node, cluster_id
+ RETURN node.id as node_id, cluster_id
+   ORDER BY node_id ASC;
+```
+
+### `set_clusters( n_clusters, embedding_property, cluster_property, init, n_init, max_iter, tol, algorithm, random_state)`
+Procedure sets for each node to which cluster it belongs to by writing cluster id  to `cluster_property`.
+
+#### Input:
```

Comment:
Plase check the changes I made in a similar block above, and apply the same changes here